Jenkinsfile

```diff
@@ -490,6 +506,107 @@ az aks create                      \
+    def eksVersions = ["1.11"]
+    for (x in eksVersions) {
+        def kversion = x // local bind required because closures
+        def platform = "eks-${kversion}"
+        platforms[platform] = {
+            stage(platform) {
+                node(label) {
+                    withCredentials([[
+                        $class: 'AmazonWebServicesCredentialsBinding',
+                        credentialsId: 'aws-eks-kubeprod-jenkins',
+                        accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                        secretKeyVariable: 'AWS_SECRET_ACCESS_KEY',
+                    ]]) {
+                        withEnv([
+                            "AWS_DEFAULT_REGION=eu-central-1",
+                        ]) {
+                            withGo() {
+                                dir('src/github.com/bitnami/kube-prod-runtime') {
+                                    def clusterName = ("${env.BRANCH_NAME}".take(8) + "-${env.BUILD_NUMBER}-" + UUID.randomUUID().toString().take(5) + "-${platform}").replaceAll(/[^a-zA-Z0-9]+/, '-').toLowerCase()
+                                    def dnsPrefix = "${clusterName}"
+                                    def adminEmail = "${clusterName}@${parentZone}"
+                                    def dnsZone = "${dnsPrefix}.${parentZone}"
+
+                                    try {
+                                        runIntegrationTest(platform, "eks --user-pool-id=eu-central-1_P7sIQ92Cd --config=${clusterName}-autogen.json --dns-zone=${dnsZone} --email=${adminEmail}", "--dns-suffix ${dnsZone}") {
+                                            // Create EKS cluster: requires AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment
+                                            // variables to reference a user in AWS with the correct privileges to create an EKS
+                                            // cluster: https://github.com/weaveworks/eksctl/issues/204#issuecomment-450280786
+                                            container('eksctl') {
+                                                sh "eksctl create cluster --name ${clusterName} --version ${kversion} --node-type m5.large --nodes 3 --nodes-min 3 --nodes-max 3 --timeout 60m --kubeconfig \$KUBECONFIG --tags 'platform=${platform},branch=${BRANCH_NAME},build=${BUILD_URL}'"
+                                                withEnv(["PATH+KTOOL=${tool 'kubectl'}"]) {
+                                                    sh "kubectl create clusterrolebinding cluster-system-anonymous --clusterrole cluster-admin --user system:anonymous"
+                                                }
+                                                waitForRollout("kube-system", 30)
+                                            }
+
+                                            // NB: writeJSON doesn't work without approvals(?)
+                                            // See https://issues.jenkins-ci.org/browse/JENKINS-44587
```

Comment:
Not a relevant comment here - this isn't attempting to write a JSON file.

Reply:
Done.